rhacm/policies/AC-Access_Control/component.yml

```
@@ -0,0 +1,692 @@
+        Kubernetes configuration.
+    - key: i.1
+      text: |-
+        AC-2(i.1) Authorizing access to the information system based on a valid access
+        authorization is an organizational control outside the scope of Red Hat Advanced Cluster
+        Management for Kubernetes configuration.
+    - key: i.2
+      text: |-
+        AC-2(i.2) Authorizing access to the information system based on intended system usage is
+        an organizational control outside the scope of Red Hat Advanced Cluster Management for
+        Kubernetes configuration.
+    - key: i.3
+      text: |-
+        AC-2(i.3) Authorizing access to the information system based on other attributes as
+        required by the organization or associated missions/business functions is an organizational
+        control outside the scope of Red Hat Advanced Cluster Management for Kubernetes
+        configuration.
+    - key: j
+      text: |-
+        AC-2(j) Reviewing accounts for compliance with account management requirements is an
+        organizational control outside the scope of Red Hat Advanced Cluster Management for
+        Kubernetes configuration.
+    - key: k
+      text: |-
+        AC-2(k) Establishing a process for reissuing shared/group account credentials (if deployed)
+        when individuals are removed from the group is an organizational control outside the scope
+        of Red Hat Advanced Cluster Management for Kubernetes configuration.
+
+# NIST Control URL:
+#   https://nvd.nist.gov/800-53/Rev4/control/AC-2?baseline=moderate#enhancement-1
+- control_key: AC-2 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |-
+        Employing automated mechanisms to support the management of information
+        system accounts is an organizational control outside the scope of Red Hat
+        Advanced Cluster Management for Kubernetes configuration.
+
+        Red Hat Advanced Cluster Management for Kubernetes relies on OpenShift
+        components that support automated management through monitoring of
+        accounts and system activity by logging of related information.
+
+# AC-2(2) NOTES:
+#       The customer will be responsible for automatically removing or
+#       disabling emergency and temporary accounts within the required
+#       timeframe. A successful control response will need to address
+#       all of the procedures and mechanisms involved in disabling these
+#       accounts.
+#
+# NIST Control URL:
+#   https://nvd.nist.gov/800-53/Rev4/control/AC-2?baseline=moderate#enhancement-2
+- control_key: AC-2 (2)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |-
+        Red Hat Advanced Cluster Management for Kubernetes relies on
+        OpenShift for account management.
+
+        This control is applicable to the identity service provider
```

```suggestion
        This control is applicable to the OpenShift identity service provider
```

@carlosmmatos make this change so that it is clear about the identity service provider. Make sure to check for all occurrences of it.